### ISSUE TITLE
Linearised Dependencies

### ISSUE THREAD

**Issue by JustCunn** [stanfordnlp/CoreNLP#1074]
Hi,

I'm currently working on a NMT model that takes in linearised constituency trees and uses them. I was wondering if there was anything similar for dependencies where one sentence (in its 'tree' form) is linearised to just one line.

Justin

**Comment by J38** [comment]
Could you display a sample tree? Do you just mean printing the PTB format on one line (spaces instead of newlines)?

**Comment by JustCunn** [comment]
Yes, similar to the `oneline` flag for Constituency trees. Unless it isn't a thing for Dependency trees.

So, of course, here is one of my normal dependency trees:
(sentence is "Citizens Information is also available")
```
compound(information-2, citizens-1)
nsubj(available-5, information-2)
cop(available-5, is-3)
advmod(available-5, also-4)
root(ROOT-0, available-5)
```
But my NMT task is based on seq2seq so I need something similar to:
```
compound(information-2, citizens-1) nsubj(available-5, information-2) cop(available-5, is-3) advmod(available-5, also-4) root(ROOT-0, available-5)
```

Thanks
